app/src/main/res/layout/activity_onboarding.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Okay a little complicated (nesting a ConstraintLayout inside a CoordinatorLayout) but if it works that's fine :)